homeassistant/components/almond/__init__.py

```diff
@@ -247,6 +247,11 @@ async def async_get_access_token(self):
 class AlmondAgent(conversation.AbstractConversationAgent):
     """Almond conversation agent."""
 
+    _attr_attribution = {
```

Comment:
It is not based on Entity classes so does not have entity attribute

Reply:
You're right, this can be closed.